src/components/structural/Textures.js

```diff
@@ -0,0 +1,28 @@
+let TexturePack = [
+    {title:"bricks",url:"img/textures/bricks.jpg"},

```

Comment:
Small edit, but can these have a slash at the start. Like "/img/textures/bricks.jpg"?